src/services/generate-default-data.ts

```diff
@@ -14,7 +14,8 @@ export default function generateDefaultData(
       case 'checkboxes':
       case 'select':
       case 'autocomplete':
-      case 'radio': {
+      case 'radio':
+      case 'compliance': {
```

Comment:
I dont think this will work. `compliance` element submission data is not the same as a radio button.

Reply:
true